docs/get-started/getting-started-on-prem.md

```diff
@@ -296,6 +299,32 @@ $SPARK_HOME/bin/spark-shell \
        --jars ${SPARK_CUDF_JAR},${SPARK_RAPIDS_PLUGIN_JAR}
 ```  
 
+### MIG GPU on YARN
+Using MIG enabled GPUs on YARN requires enabling YARN GPU scheduling, cgroups, and using NVIDIA Docker runtime v2.
+The way to set this up depends on the version of YARN and the version of Spark.
+It is important to note that CUDA 11 only supports enumeration of a single MIG instance. This means that using
+any MIG device on YARN means only 1 GPU per container is allowed. See the limitations section in the documentation
+referred to below for the specific YARN version you are using.
+
+#### YARN version 3.3.0+
+YARN version 3.3.0 and newer support a pluggable device framework which allows adding support for MIG devices via a plugin.
+See [NVIDIA GPU Plugin for YARN with MIG support](https://github.com/NVIDIA/spark-rapids-examples/blob/branch-21.10/hadoop/device-plugins/gpu-mig/README.md).
+If you are using that plugin with a Spark version older than 3.2.1 and/or specifying the resource as `nvidia/miggpu` you will also need to specify the config:
+
+```shell
+--conf spark.rapids.gpu.resourceName=nvidia/miggpu
+```
+
+This tells the RAPIDS Accelerator for Spark plugin to look for the Spark GPU resource assigned to it using the name `nvidia/miggpu`.
+If you are using the Spark config `spark.yarn.resourceGpuDeviceName` and using the normal `gpu` Spark resource name, this is not required.
+
+#### YARN version 3.1.2 until 3.3.0
+If you are using YARN version from 3.1.2 up until 3.3.0, it requires making modifications to YARN and deploying
+a version that adds support for MIG to the built-in YARN GPU resource plugin.
+
+See [NVIDIA Support for GPU for YARN with MIG support for YARN 3.1.2 until YARN 3.3.0](https://github.com/NVIDIA/spark-rapids-examples/blob/branch-21.10/hadoop/resource-types/gpu-mig/README.md)
```

Comment:
This link is broken, I think it needs to be:
```suggestion
See [NVIDIA Support for GPU for YARN with MIG support for YARN 3.1.2 until YARN 3.3.0](https://github.com/NVIDIA/spark-rapids-examples/blob/branch-21.10/hadoop/device-plugins/gpu-mig/README.md)
```

Reply:
actually that is being added under the PR referenced in the description

Reply:
Ah sorry. missed that the other docs were still in flight.